source/agora/test/Flash.d

```diff
@@ -419,3 +419,120 @@ unittest
     writefln("alice closing tx: %s", alice.getClosingTx(alice_bob_chan_id));
     assert(block12.txs[0] == alice.getClosingTx(alice_bob_chan_id));
 }
+
+/// Test indirect channel payments
```

Comment:
Should the comment be updates? Seems like copypasta. :)

I'll merge now and you can fix it.later.

Reply:
Or self-merge when ready!

Reply:
Thanks, updated.